lib/PerfMeter.js

```diff
@@ -139,7 +139,7 @@ PerfMeter.prototype = {
 
   displayItem(tabId, item) {
     if (this._active) {
-      console.info(`${tabId} ${item.start} ${item.tag}${item.data ? ":" + item.data : ""} ${item.delta || ""}`);
+      console.info(`${tabId} ${item.start} ${item.tag}${item.data ? ":" + item.data : ""} ${item.delta || ""}`); // eslint-disable-line no-console
```

Comment:
Re: `${item.data ? ":" + item.data : ""}`,

Do we need all that, or can we short circuit like: `${item.data || ""}` (a la `item.delta`)